tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
-    "lib": ["es5", "es2015", "dom"],
+    "lib": ["es2020", "dom"],
+    "target": "es2015",
```

Comment:
When undefined, typescript uses `es3` as the default value, which is very inefficient